client/stylesheets/puzzle.scss

```diff
@@ -367,8 +367,53 @@ table.puzzle-list {
     margin-left: 4px;
     position: relative;
 
+    .icon {
+      font-size: 12px;
+      width: 16px;
+      text-align: center;
+      color: $danger;
+      height: 50%;
+    }
+
+    $mutedTop: -4px;
+    $mutedRight: -2px;
+    $deafenDiff: 2px;
```

Comment:
It would be simpler to have the border all the time, except I'd have to frob the spectrum width for it to still go edge to edge -- it looks kind of funky otherwise.

Reply:
Yeah, my preference would tend towards that. It still has an awkward offset but eliminates the state dependence.